tests/main/interfaces-network-bind/task.yaml

```diff
@@ -60,3 +54,8 @@ execute: |
     echo "Then the service is not accessible by a client"
     response=$(nc -w 2 localhost "$PORT" < $REQUEST_FILE)
     [ "$response" = "" ]
+
+    # This snap is removed because it generates thousands of DENIALS in the journal. Most of those
+    # are sent after the journalctl cursor for following test is determined producing errors while
+    # preparing the test.
+    snap remove network-bind-consumer
```

Comment:
Won't that pollute the logs if the test fails before this line is reached and the snap is not cleaned up?

Reply:
Thanks for reviewing, I changed the test to check the re-connection at the end and moved the removal to the restore as it was before.